pkg/status/combinedstatus-resolver.go

```diff
@@ -102,6 +103,8 @@ type CombinedStatusResolver interface {
 	//
 	// The returned pointers are expected to be read-only.
 	ResolutionExists(name string) (string, util.ObjectIdentifier, bool)
+
+	ResolutinosForBinding(bdgName string) map[util.ObjectIdentifier]*combinedStatusResolution
```

Comment:
(A) Typo in this method's name: "Resolutinos" instead of "Resolutions".

(B) When a mutable data structure is returned, you need a comment about whether caller and/or anyone else can mutate it after the return.

Reply:
Fixed.